src/error.rs

```diff
@@ -135,6 +135,29 @@ pub enum ConvertError<A, S, V> {
     Validity(V),
 }
 
+impl<Src, Dst: ?Sized + Unaligned, S, V> ConvertError<AlignmentError<Src, Dst>, S, V> {
+    // TODO: Bikeshed this name
+    pub fn recall_aligned(self) -> ConvertError<Infallible, S, V> {
+        match self {
+            Self::Alignment(e) => ConvertError::Alignment(e.into_infallible()),
+            Self::Size(e) => ConvertError::Size(e),
+            Self::Validity(e) => ConvertError::Validity(e),
+        }
+    }
+}
+
+impl<Src, Dst: ?Sized + Unaligned, S, V> Into<ConvertError<Infallible, S, V>>
```

Comment:
Why are we implementing `Into` here instead of `From`?

Reply:
Done